edd-status-board.php

```diff
@@ -9,10 +9,19 @@
 License: GPLv2 or later
 */
 
-if ( class_exists( 'Easy_Digital_Downloads' ) && version_compare( EDD_VERSION, '2.4', '>=' ) ) {
-	add_filter( 'edd_api_valid_query_modes', 'edd_statusboard_mode' );
-	add_filter( 'edd_api_output_data', 'edd_statusboard_output', 10, 2 );
-	add_action( 'show_user_profile',   'edd_statusboard_profile_endpoint_display' );
+add_action( 'init', 'edd_statusboard_init' );
```

Comment:
```suggestion
add_action( 'plugins_loaded', 'edd_statusboard_init' );
```

Just suggesting this because `plugins_loaded` happens a little earlier and that's when EDD loads. `init` would probably be fine, but I think `plugins_loaded` is better.